user_guide_src/source/libraries/validation.rst

```diff
@@ -1082,6 +1082,12 @@ max_dims                Yes         Fails if the maximum width and height of an
                                     the width, and the third is the height. Will
                                     also fail if the file cannot be determined
                                     to be an image.
+min_dims                Yes         Fails if the maximum width and height of an  ``min_dims[field_name,300,150]``
+                                    uploaded image not meet values. The first
+                                    parameter is the field name. The second is
+                                    the width, and the third is the height. Will
+                                    also fail if the file cannot be determined
+                                    to be an image.
```

Comment:
Please add `(This rule was added in v4.6.0.)` at the end.

Reply:
Changed